cvpr2017/python/superuser-init.py

```diff
@@ -0,0 +1,207 @@
+        Program_Chairs = openreview.Group(COCHAIRS,
+            readers     = [CONFERENCE, COCHAIRS],
+            writers     = [CONFERENCE],
+            signatures  = [CONFERENCE],
+            signatories = [COCHAIRS],
+            members     = [])
+        groups.append(Program_Chairs)
+
+    if overwrite_allowed(REVIEWERS):
+        reviewers = openreview.Group(REVIEWERS,
+            readers     = [CONFERENCE, COCHAIRS, REVIEWERS],
+            writers     = [CONFERENCE],
+            signatures  = [CONFERENCE],
+            signatories = [REVIEWERS],
+            members     = []) #more to be added later, from the list of Program_Committee members
+        groups.append(reviewers)
+
+    ## Post the groups
+    for g in groups:
+        print "Posting group: ",g.id
+        client.post_group(g)
+
+    client.post_group(client.get_group('host').add_member(CONFERENCE))
+
+
+    #########################
+    ##  SETUP INVITATIONS  ##
+    #########################
+    call(["node", "../../scripts/processToFile.js", "../process/submissionProcess.template", "../process"])
+
+    invitations = []
+
+    ## Create the submission invitation, form, and add it to the list of invitations to post
+    submission_invitation = openreview.Invitation(CONFERENCE,
+        'Submission',
+        readers = ['everyone'],
+        writers = [CONFERENCE],
+        invitees = ['~'],
+        signatures = [CONFERENCE],
+        duedate = EIGHT_PG_TIMESTAMP_DUE,
+        process = '../process/submissionProcess.js')
+
+    submission_invitation.reply = {
+        'forum': None,
+        'replyto': None,
+        'readers': {
+            'description': 'The users who will be allowed to read the above content.',
+            'values': ['everyone']
+        },
+        'signatures': {
+            'description': 'How your identity will be displayed with the above content.',
+            'values-regex': '~.*'
+        },
+        'writers': {
+            'values-regex': '~.*'
+        },
+        'content': {
+            'title': {
+                'description': 'Title of paper.',
+                'order': 1,
+                'value-regex': '.{1,250}',
+                'required':True
+            },
+            'authors': {
+                'description': 'Comma separated list of author names.',
+                'order': 2,
+                'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
+                'required':True
+            },
+            'authorids': {
+                'description': 'Comma separated list of author email addresses, in the same order as above. Be sure each email address is linked to the each author profile.',
+                'order': 3,
+                'values-regex': "([a-z0-9_\-\.]{2,}@[a-z0-9_\-\.]{2,}\.[a-z]{2,},){0,}([a-z0-9_\-\.]{2,}@[a-z0-9_\-\.]{2,}\.[a-z]{2,})",
+                'required':True
+            },
+            'paper length': {
+                'description': 'Type of paper.',
+                'order': 0,
```

Comment:
I wanted to put it first so that it was very clear that there are separate tracks

I asked stratis to take a look and let me know what he thought